test/read_mixin_test.rb

```diff
@@ -128,6 +128,12 @@
       end
     end
 
+    it "finds cached when searching by column" do
+      Post.where(title: 'no_comments').first.title.must_equal 'no_comments'
+      Post.expects(:find_by_sql_without_kasket).never
+      Post.where(title: 'no_comments').first.title.must_equal 'no_comments'
```

Comment:
Hmm, it’s hard to be explicit about testing the implicit order. Should we add a comment saying that this query will test `order by id asc`? Or an assertion about `to_sql.ends_with?("order by id asc")`?